xds/src/main/java/io/grpc/xds/ClientXdsClient.java

```diff
@@ -2537,8 +2537,17 @@ void onError(Status error) {
         respTimer.cancel();
         respTimer = null;
       }
+
+      // Include node ID in RPC failure status messages that originate from XdsClient.
+      // We expect all watchers to propagate the description to the channel, and expose it
```

Comment:
Nit: we recently made a fix to *not* propagate the error-code from the xdsClient to the RPC caller. That error-code to the RPC caller should always be UNAVAILABLE. So just want to make sure that that piece of code does not remove this error-description. And it might be useful to mention in this comment that we propagate the error description but *not* the error code.

-